engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -289,7 +289,11 @@ private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: Commo
   }
 
   private def extractProperty(e: PropertyOrFieldReference, t: TypingResult): ValidatedNel[ExpressionParseError, TypingResult] = t match {
-    case Unknown => Valid(Unknown)
+    case Unknown =>
```

Comment:
this is for the properties (e.g. fields), we also need check for methods - e.g. in TypeMethodReference

Reply:
Haven’t seen this, check for methods in TypeMethodReference is already implemented.